src/modules/campath/mod.rs

```diff
@@ -72,6 +72,20 @@ Offsets time values in second of each campath entry. Offset can be negative. Def
 ",
 );
 
+static BXT_CAMPATH_ROTATE: CVar = CVar::new(
+    b"bxt_campath_rotate\0",
+    b"0\0",
+    "\
+Rotates all camera points around origin by Z up axis. HLAE CAM format is mainly for Source. But, Source horizontal rotation is slightly different from GoldSrc.
```

Comment:
```suggestion
Rotates all camera points around origin by Z up axis.

HLAE CAM format is mainly for Source. But, Source horizontal rotation is slightly different from GoldSrc.
```

I'm also not sure what "Rotates all camera points around origin by Z up axis." means. Shouldn't this say something like "Rotate all camera points about (some specific axis) by this angle."?

Reply:
It means exactly what you think. I just can't quite unwrap the technical details so people may understand why they need to do this.

Reply:
Maybe "Rotates all camera points about the Z-up axis at the origin by this angle in degrees."?